graphviz.c: In function ‘dot_neato_layout_cb’:
graphviz.c:270: error: request for member ‘x’ in something not a structure
or union
graphviz.c:271: error: request for member ‘y’ in something not a structure
or union

http://code.google.com/p/ggobi-documentation/issues/detail?id=29

--- origsrc/ggobi-2.1.8/plugins/GraphLayout/graphviz.c	2008-02-13 07:49:38.000000000 -0600
+++ src/ggobi-2.1.8/plugins/GraphLayout/graphviz.c	2010-09-29 00:15:03.020775900 -0500
@@ -267,8 +267,8 @@ void dot_neato_layout_cb (GtkWidget *but
       m = visible[i];
       name = (gchar *) g_array_index (d->rowlab, gchar *, m);
       node = agfindnode (graph, name);
-      pos[i][0] = (gdouble) ND_coord_i(node).x;
-      pos[i][1] = (gdouble) ND_coord_i(node).y;
+      pos[i][0] = (gdouble) ND_coord(node).x;
+      pos[i][1] = (gdouble) ND_coord(node).y;
     }
 
   } else if (layout_type == TWOPI_LAYOUT) {
